main.c

```diff
@@ -466,13 +475,41 @@ static void setup_property_changed_listener(void) {
 }
 #endif
 
+static void wl_seat_capabilities_handler(
+    void *data,
+    struct wl_seat *seat,
+    uint32_t capabilities
+) {
```

Comment:
Style:

```c
static void wl_seat_capabilities_handler(void *data, struct wl_seat *seat,
		uint32_t capabilities) {
	…
}
```

(Applies to other function definitions too)

Reply:
Bump

Reply:
whoops